### DISCUSSION TITLE
pages build and deployment error

### DISCUSSION THREAD

**Discussion by ghost** [alshedivat/al-folio#952] [Q&A]
Hello, I am trying to create a site, but this error is given, what should I do?

**Comment by cmoralesmx** [comment]
Where are you seeing the error?

If you mean you are pushing your changes to a GitHub hosted repository and you are seeing this message in the actions section, it means you are using the default GitHub pages build-and-deploy action and not the custom al-folio action.
You need to follow the [Deployment](https://github.com/alshedivat/al-folio#deployment) instructions and pay special attention to the section `To enable automatic deployment`
